packages/aws-cdk-lib/aws-opensearchservice/test/domain.test.ts

```diff
@@ -1789,6 +1795,88 @@ each(testedOpenSearchVersions).describe('custom endpoints', (engineVersion) => {
   });
 });
 
+each(testedOpenSearchVersions).describe('TLS security policy', (engineVersion) => {
+  test('defaults to TLS 1.2 when tlsSecurityPolicy is not specified', () => {
+    new Domain(stack, 'Domain', {
+      version: engineVersion,
+    });
+
+    Template.fromStack(stack).hasResourceProperties('AWS::OpenSearchService::Domain', {
+      DomainEndpointOptions: {
+        TLSSecurityPolicy: 'Policy-Min-TLS-1-2-2019-07',
+      },
+    });
+  });
+
+  test('defaults to TLS 1.2 when enforceHttps is true but tlsSecurityPolicy is not specified', () => {
```

Comment:
The test above should already cover all the scenarios where the policy is not specified (regardless of whether `enforceHttps` is true or not).